openapi/src/parts/controlled_gcp_ai_notebook_instance.yaml

```diff
@@ -130,6 +130,27 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/gcp/ai-notebook-instances/generateName:
+    parameters:
+    - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: Generate a cloud native controlled Gcp bucket name
+      operationId: getCloudNameFromAiNotebookInstanceName
```

Comment:
AiNotebookInstanceName seems confusing as it is the cloud name to me.

Can you rename this to generateAiNotebookCloudInstanceId

Reply:
Sure. the names follows the Dan's propose in the design doc. But I am ok with both Idea.